vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -174,10 +174,42 @@ def save_containers_inventory(container_group, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:containers, container_group, hashes, deletes, [:ems_ref])
+    hashes.each do |h|
+      h[:container_image_id] = h[:container_image][:id]
```

Comment:
It's filled when you use `save_inventory_multi` on the `:container_image` hashes.